Tests/conf.json

```diff
@@ -21,6 +21,17 @@
     "testTimeout": 160,
     "testInterval": 20,
     "tests": [
+        {
+            "playbookID": "Test Playbook - Impossible Traveler Response",
+            "integrations": [
+                "VirusTotal (API v3)",
+                "Microsoft Graph"
```

Comment:
do you need whois as well for geolocation?

Reply:
Also, I see that you are referring to AD here - but you didn't mention it in this file 
![image](https://github.com/demisto/content/assets/88268646/eb8ecff4-4c41-4436-b5d3-7b8c92cc91e8)
![image](https://github.com/demisto/content/assets/88268646/03491f84-83de-43af-b85c-bd882fe4622e)

Reply:
removed the AD mentioned in the 'printerrorentry' tasks.